Imported v 0.82 changes

Spiralo-Idioide opened the pull request:
Just replaced all libraries with the ones from the v 0.82 that can be downloaded at the website.
I didn't check it and I'm sure there are things that need not to be merged.

EDIT :
I think I checked everything now. Also I changed 'userHomePath' to 'GP Mod Folder' but that's more some kind of personnal preference.

Spiralo-Idioide committed:
Imported v 0.82 changes

Spiralo-Idioide committed:
Update AuthoringSpecs.gp

Spiralo-Idioide committed:
Update FilePicker.gp

Spiralo-Idioide commented on runtime/lib/version.gp:
```diff
@@ -1 +1 @@
-to libraryVersion { return 'v081' }
+to libraryVersion { return 'v082' }
```
Kept v082 because technically it is the version of the library

Spiralo-Idioide committed:
Update morphicFramework.gp

Spiralo-Idioide committed:
Update morphicFramework.gp

Spiralo-Idioide committed:
Update morphicFramework.gp

Spiralo-Idioide committed:
Update Project.gp

Spiralo-Idioide committed:
Update FilePicker.gp

Spiralo-Idioide committed:
Update FilePicker.gp

Spiralo-Idioide committed:
Update ProjectEditor.gp

Spiralo-Idioide committed:
Update ProjectEditor.gp

Spiralo-Idioide committed:
Update Stage.gp

Spiralo-Idioide committed:
Update morphicFramework.gp

Spiralo-Idioide committed:
Update Stage.gp

Spiralo-Idioide commented on runtime/lib/Project.gp:
```diff
@@ -685,29 +685,25 @@ method removeClassFromPages Project aClass {
 
```
This one shouldn't be merged but I don't know how to easily get rid of it. Still don't know how to use Git

Spiralo-Idioide committed:
Update startup.gp

Spiralo-Idioide committed:
Revert to GP mod logo added by MSandro

Spiralo-Idioide committed:
Merge branch 'v-082' of https://github.com/Spiralo-Idioide/GP-Mods.git into v-082

callowaysutton commented:
I thought v082 just had a change in the executable

Is this just converting everything so that it says GP Mod instead of GP everywhere?

Spiralo-Idioide commented:
@callowaysutton 
> I thought v082 just had a change in the executable

I also though that but by using v-081 I realized a huge bug that I had never encountered before when using v-082. 
It's this :
![ElseIf](https://framapic.org/LS5bRO2XJCjP/NbqHlTLw5Lh7.gif)

> Is this just converting everything so that it says GP Mod instead of GP everywhere?

No, but I think having a folder called 'GP Mod' might be better.

callowaysutton commented:
That is weird lol

Spiralo-Idioide committed:
Fixed lines

For some reasons some lines were removed when I pushed the file, that's fixed now.

Spiralo-Idioide committed:
Line fix

Last commit didn't work, I don't really understand why so I'm trying again by just removing a dot in the first comment

Spiralo-Idioide committed:
Space fix - 3

The problem is only on Github.
Sorry if the pull request now looks like nothing but I want to understand why it doesn't work
